go/vt/vttablet/tabletserver/tx_engine_test.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/youtube/vitess/go/vt/vttablet/tabletserver/tabletenv"
 
+	"github.com/youtube/vitess/go/vt/proto/query"
```

Comment:
This should be renamed to `querypb "github...`.

I have a patch you can apply to goimports to do this automatically: https://gist.github.com/sougou/2255bc77bf8eaa45eadcc44f8832514f